docs/contributing/contributing.rst

```diff
@@ -84,6 +71,27 @@ For more information, see the `pytest documentation`_.
 
 .. _test_env_vars:
 
+Running Integration Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These tests are generally only run using in our CI workflow due to their resource requirments and cost. However, they
+can be made the available for local testing:
+
+ - Running tests that make use of Docker (e.g. autoscaling tests and Docker tests)
+   require an appliance image to be hosted. This process first requires :ref:`quaySetup`.
+   Then to build and host the appliance image run the ``make`` targets ``docker``
+   and ``push_docker`` respectively. ::
+
+        $ make docker; make push_docker
```

Comment:
`make push_docker` already includes `make docker`.  Look at the Makefile: https://github.com/DataBiosphere/toil/blob/master/Makefile#L240

Only `make push_docker` should be necessary.